adafruit_irremote.py

```diff
@@ -66,7 +71,7 @@ class IRNECRepeatException(Exception):
     """Exception when a NEC repeat is decoded"""
 
 
-def bin_data(pulses):
+def bin_data(pulses) -> List[List]:
```

Comment:
Lets add the type for `pulses` as well. With this first pass we are trying to knock out all function arguments and returns.

From the docstrings below it looks like `pulses` is expected to be a List.